src/Work/Formulas.ts

```diff
@@ -119,7 +120,18 @@ export const calculateCompanyWorkStats = (
   const gains = scaleWorkStats(
     multWorkStats(
       {
-        money: companyPosition.baseSalary * company.salaryMultiplier * bn11Mult * currentNodeMults.CompanyWorkMoney,
+        money:
+          companyPosition.baseSalary *
+          company.salaryMultiplier *
+          bn11Mult *
+          currentNodeMults.CompanyWorkMoney *
+          (Player.augmentations
+            .map((aug: PlayerOwnedAugmentation) => {
+              return aug.name;
+            })
+            .filter((aug) => aug == AugmentationName.EmpathySuppressor).length == 1
```

Comment:
This all can be replaced with `Player.hasAugmentation(AugmentationName.EmpathySuppressor, true)`

Your current code iterates the entire augmentations array 2 times over, where hasAugmentation or using your own .find or .some will only iterate through it once partially (stopping as soon as a match is found).